input/data/2013/2013ApJ...779...88A/tev-000014.yaml

```diff
@@ -0,0 +1,21 @@
+source_id: 14
+reference_id: 2013ApJ...779...88A
+telescope: veritas
+
+# Orbital period
+orbit:
+    # zero phase for orbital period from Gregory (2002)
+    mjd0: 43366.3     #
+    period: {val: 26.4960 }
+
+# (spectral fit for phases 0.5-0.8)
+spec:
+  model:
+    type: pl
+    parameters:
+      mjd: {min: 55911.0, max: 55497}
```

Comment:
`mjd` and `phase` are not spectral model parameters. they should be listed directly under the `spec` key, as you have it in other files. See https://github.com/gammapy/gamma-cat/blob/master/input/schemas/dataset_source_info.schema.yaml#L241